src/engine/engine.cpp

```diff
@@ -169,29 +169,33 @@ class EngineImpl : public Engine
 			if (lua_isstring(L, -1))
 			{
 				const char* str = lua_tostring(L, -1);
-				desc.set(cmp, -1, InputBlob(str, stringLength(str)));
+				InputBlob input_blob(str, stringLength(str));
+				desc.set(cmp, -1, input_blob);
```

Comment:
`IPropertyDescriptor::set` takes lvalue reference (which temporary InputBlob is not).